owa/app/js/home/home.html

```diff
@@ -1,3 +1,6 @@
+<div>
+    <a class="button" href="#/mapping"><i class="icon-plus"></i>&#160;Create Mapping</a>
```

Comment:
@AdamGrzybkowski it looks fine to me. If you do not get a response from him, go ahead and merge. :) Am more inclined to merging and making followup fixes than having a pull request open for weeks! :)